app-editors/joe/joe-4.4.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Terminal-based text editor"
+DESCRIPTION="Joe's Own Editor is a full-featured text editor with minimal requirements. \
+It is fully customizable (Emacs, pico or WordStar are preset). Is also has multiple \
+windows, hexadecimal edit mode for binary files or syntax highlighting for more than 40 languages. \
+It also supports mouse for both text and window manipulation."
+HOMEPAGE="http://joe-editor.sourceforge.net/"
+LICENSE="GNU GPL v1"
+COPYRIGHT="1992, 2004, 2006 Joseph H. Allen"
+SOURCE_URI="https://downloads.sourceforge.net/project/joe-editor/JOE sources/joe-4.4/joe-4.4.tar.gz"
+CHECKSUM_SHA256="a5704828bbca29acb9e200414fef522c66cdf9ce28150f402d6767da43460979"
+SOURCE_DIR="joe-4.4"
+REVISION="1"
+PATCHES="0001-Fix-gcc2-build.patch"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+
+PROVIDES="
+	joe = $portVersion compat >= 0
+	cmd:joe = $portVersion compat >= 0
```

Comment:
yeah I had 4 in there but dropped it to 0 in an attempt to get Haikuporter to give me some sort of useful error message.  Didn't help.